### ISSUE TITLE
Feature: Update existing variables 

### ISSUE THREAD

**Issue by bradym** [profclems/glab#899]
**Describe the feature or problem you'd like to solve**
 The gitlab API allows for updating existing CI variables, but this has not yet been implemented in glab.

**Propose a Solution**
Given how similar the set and update api endpoints are, I think it would make sense to add an update flag to the `set` command that would call new `UpdateGroupVariable` or `UpdateProjectVariable` functions. 

Then you'd update a variable like this:

```
glab variable set VAR_NAME newValue --update
glab variable set VAR_NAME newValue --update --group myGroup
```
@profclems what do you think? I'd like your thoughts on this before starting work on this one.

**Comment by bradym** [comment]
As mentioned on #902 I decided to implement as a new command instead as it was both easier and more consistent with other commands.

**Comment by clemsbot** [comment]
This issue has been automatically marked as stale because it has not had recent activity. We haven't had the time to address it yet, but we want to keep it open. This message is just a reminder for us to help triage issues.

**Comment by bradym** [comment]
This was fixed in #902 which has been merged into trunk.
